scripts/ci-integration.sh

```diff
@@ -66,7 +66,23 @@ prepare_crd_yaml() {
 }
 
 create_bootstrap() {
-   kind create cluster --name "${BOOTSTRAP_CLUSTER_NAME}"
+   cat <<EOF > /tmp/integration.cluster
```

Comment:
We may want to consider using `mktemp` combined with a `trap` function to remove it... something I've done in the past.

Reply:
Done, ptal